### DISCUSSION TITLE
How to join Prisma channels at prisma.slack.com?

### DISCUSSION THREAD

**Discussion by MurzNN** [prisma/prisma#7734] [Q&A] [answered]
I registered at prisma.slack.com and successfully logged in, but can't understand how to join most of Prisma channels?
In "Add channels" list I see only the "mongodb-early-access" channels, and that's all:
![image](https://user-images.githubusercontent.com/336662/122519574-48435a00-d01b-11eb-90eb-011275d958f7.png)

**Comment by ryands17** [comment]
@MurzNN 👋 

That's strange! I clicked on **Browse Channels** and I get a list of all the channels in the workspace

![image](https://user-images.githubusercontent.com/19697099/122520526-53a18000-d031-11eb-9829-012716b60c0a.png)

![image](https://user-images.githubusercontent.com/19697099/122520570-60be6f00-d031-11eb-898e-7dca58032abe.png)

@janpio do you have any idea?

**Reply by janpio** [reply]
I pinged someone who might know what is happening here.

**Comment by MurzNN** [comment, marked as answer]
Suddenly all channels are go back, without any actions from my side... Magic! :mage_man:

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
